v1.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func v1(w http.ResponseWriter, r *http.Request) {
+	conn, err := upgrader.Upgrade(w, r, nil)
+
+	if err != nil {
+		log.Println("Can't upgrade to WebSocket connection.")
+		return
+	}
+
+	defer conn.Close()
+
+	for {
+		action := Action{}
+		err = conn.ReadJSON(&action)
+
+		if err != nil {
+			log.Println("Error reading JSON.")
+			writeError(conn, 2001)
+			continue
+		}
+
+		if action.Type == "" {
+			log.Println("Action Type is required.")
+			writeError(conn, 3001)
+			continue
+		}
+
+		if action.Payload == nil {
+			log.Println("Action Payload is required.")
+			writeError(conn, 3001)
+			continue
+		}
+
+		switch action.Type {
+		case "VERIFICATION_VERIFY_PHONE_NO":
+			countryCode, ok := action.Payload["country_code"].(float64)
+
+			if !ok {
+				log.Println("country_code is required in Payload.")
+				writeError(conn, 3001)
+				break
+			}
+
+			phoneNumber, ok := action.Payload["phone_number"].(float64)
+
+			if !ok {
+				log.Println("phone_number is required in Payload.")
+				writeError(conn, 3001)
+				break
+			}
+
+			b, err := json.Marshal(map[string]interface{}{
+				"api_key":      twilioAPIKey,
+				"country_code": int(countryCode),
+				"phone_number": int(phoneNumber),
+				"via":          "sms",
+			})
+
+			if err != nil {
+				log.Println(err)
```

Comment:
@cglotr Why don't we writeError here as well?
```suggestion
				log.Println(err)
				writeError(conn, xxxx)
```
Same goes to line 80, 88, 101. What do you think?

Reply:
Yeah, I agree. But for *101*, the server will send a failure action `VERIFICATION_VERIFICATION_CODE_SENT_FAILED`

Reply:
Done 704c212d13e3071005a081d2b48aca5a17fb1006